float64 northing_min
float64 northing_max
float64 easting_min
float64 easting_max
bool start_mission
---
bool success
string message